examples/manila-csi-plugin/nfs/auto-topology-aware/storageclass.yaml

```diff
@@ -0,0 +1,29 @@
+# Topology constraints example:
+#
+# Let's have two Manila AZs: zone-{1..2}
+# Let's have six Nova AZs: zone-{1..6}
+#
+# Manila zone-1 is accessible from nodes in zone-1 only
+# Manila zone-2 is accessible from nodes in zone-2 only
+#
+# We're provisioning into zone-1
+# availability parameter and allowedTopologies are empty, therefore the dynamic
+# share provisioning with automatic availability zone selection takes place.
+# The "volumeBindingMode" must be set to "WaitForFirstConsumer".
```

Comment:
WaitForFirstConsumer => is this new requirement of this PR ? I think AZ related change should not change storageClass behavior like this?

Reply:
I'd say it's not a requirement for this PR, but for automatic topology awareness behavior, therefore this should should reside in the storage class parameters. Do you suggest to adjust this sentence?

Reply:
no, just want to be clear on the reason of this change :)
as long as it's reasonable I am fine with it